rpm/lua-rex.spec

```diff
@@ -0,0 +1,257 @@
+Name:           lrexlib
+Version:        2.9.0
+Release:        1%{?dist}
+Summary:        Regular expression handling library for Lua
+
+Group:          Development/Libraries
+License:        MIT
+URL:            http://lrexlib.luaforge.net/
+Source0:        lrexlib-%{version}.tar.gz
+
+%description
+Lrexlib are bindings of six regular expression library APIs (POSIX, PCRE, PCRE2,
+Oniguruma, Tre and GNU) to Lua.
+
+%package pcre
+Summary: Lua binding of PCRE library
+BuildRequires:  pcre-devel
+BuildRequires:  pkgconfig
+BuildRequires:  tarantool-devel >= 1.9.0.0
+BuildRequires:  luarocks
+Requires:       tarantool >= 1.9.0.0
+Provides:       lrexlib-pcre = %{version}
+
+%description pcre
+Binding of PCRE library
+
+%package pcre2
+Summary: Lua binding of PCRE-2 library
+BuildRequires:  pcre2
+BuildRequires:  pcre2-devel
+BuildRequires:  pkgconfig
+BuildRequires:  tarantool-devel >= 1.9.0.0
+BuildRequires:  luarocks
+Requires:       tarantool >= 1.9.0.0
+Provides:       lrexlib-pcre2 = %{version}
+
+%description pcre2
+Binding of PCRE2 library
+
+%package posix
+Summary: Lua binding of POSIX library
+BuildRequires:  pkgconfig
```

Comment:
No require of some posix regex library?

Reply:
No require. You may take a look at https://github.com/tarantool/lrexlib/blob/master/src/posix/lposix.c#L16

Reply:
Okay, it seems to be from libc:

$ equery b /usr/include/regex.h 
 * Searching for /usr/include/regex.h ... 
sys-libs/glibc-2.27-r3 (/usr/include/regex.h)